smart-contract-viem/README.md

```diff
@@ -0,0 +1,33 @@
+# Connect to Hedera Testnet via Viem
+
+This code snippets demonstrates how to connect to Hedera Testnet
+using Viem.
+
+## Code
+
+Copy the `.env.example` file to `.env`,
+then edit it to add the URL for a JSON-RPC endpoint for Hedera Testnet.
```

Comment:
additionally, you may want to mention adding a private key for an already funded account to `.env`